<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWMainWindow</class>
 <widget class="QMainWindow" name="QWMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txtEdit">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>7</y>
      <width>755</width>
      <height>385</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>778</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="separator"/>
    <addaction name="actClose"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="actCut"/>
    <addaction name="actCopy"/>
    <addaction name="actPaste"/>
    <addaction name="separator"/>
    <addaction name="actClear"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>格式</string>
    </property>
    <addaction name="actFontBold"/>
    <addaction name="actFontItalic"/>
    <addaction name="actFontUnder"/>
    <addaction name="separator"/>
    <addaction name="actToolbarLab"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actOpen"/>
   <addaction name="actClear"/>
   <addaction name="separator"/>
   <addaction name="actCut"/>
   <addaction name="actCopy"/>
   <addaction name="actPaste"/>
   <addaction name="separator"/>
   <addaction name="actFontItalic"/>
   <addaction name="actFontBold"/>
   <addaction name="actFontUnder"/>
   <addaction name="separator"/>
  </widget>
  <action name="actCut">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/cut.jfif</normaloff>:/image/img/cut.jfif</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
   <property name="toolTip">
    <string>剪切文本</string>
   </property>
  </action>
  <action name="actPaste">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/paste.jfif</normaloff>:/image/img/paste.jfif</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
   <property name="toolTip">
    <string>粘贴文本</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/copy.jfif</normaloff>:/image/img/copy.jfif</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="actFontBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/fontbold.jfif</normaloff>:/image/img/fontbold.jfif</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </action>
  <action name="actFontItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/fontItalic.jfif</normaloff>:/image/img/fontItalic.jfif</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
  </action>
  <action name="actFontUnder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/fontunder.jfif</normaloff>:/image/img/fontunder.jfif</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
  </action>
  <action name="actClose">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/close.jfif</normaloff>:/image/img/close.jfif</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/open.jfif</normaloff>:/image/img/open.jfif</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开...</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/clear.jfif</normaloff>:/image/img/clear.jfif</iconset>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="actFont">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/font.jfif</normaloff>:/image/img/font.jfif</iconset>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="actNew">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/new.jfif</normaloff>:/image/img/new.jfif</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="actToolbarLab">
   <property name="icon">
    <iconset resource="qrc.qrc">
     <normaloff>:/image/img/toolbarlab.jfif</normaloff>:/image/img/toolbarlab.jfif</iconset>
   </property>
   <property name="text">
    <string>显示工具栏文字标签</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actPaste</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCut</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>cut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actCopy</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClear</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actClose</sender>
   <signal>triggered()</signal>
   <receiver>QWMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>388</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
